CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## [v4.0.0](https://github.com/symfony/webpack-encore/releases/tag/v4.0.0)
+
+This major release makes Encore compatible with [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp) and [pnpm](https://pnpm.io/fr/).
+
+### BC Breaks
+
+* The following dependencies **must be added** in your `package.json`: `webpack webpack-cli @babel/core @babel/preset-env` (#1142 and #1150)
+* The following dependencies **must be removed** from your `package.json` and Babel configuration: `@babel/plugin-syntax-dynamic-import @babel/plugin-proposal-class-properties`, 
+since they are already included in `@babel/preset-env` (#1150) 
```

Comment:
Unless I'm mistaken, for 99% of users, this will just involve removing this line: https://github.com/symfony/recipes/blob/96d11c2e2ed2a33a796740ed47de0d6d25263fe8/symfony/webpack-encore-bundle/1.10/webpack.config.js#L49

Let's add a `diff` code block showing that 1 line being removed. I want the upgrade path to be as clear as (A) run this command and (B) remove this line